hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -139,6 +139,10 @@ public AbfsHttpOperation getResult() {
     return result;
   }
 
+  public void setMask() {
```

Comment:
Why are we creating a separate function when we are calling just the other function inside it?